app/src/androidTest/java/testUtils/MyDao.java

```diff
@@ -0,0 +1,17 @@
+package testUtils;
+
+
+import androidx.room.Dao;
+import androidx.room.Query;
+
+import com.shuttl.location_pings.data.model.entity.GPSLocation;
+
+import java.util.List;
+
+@Dao
+public interface MyDao {
```

Comment:
@nikhil-roy  How can I use This to query the database ?

Reply:
by using locations() u can get all the locations